### DISCUSSION TITLE
Time Comparison Chart Colours

### DISCUSSION THREAD

**Discussion by zhorvat-lw** [apache/superset#16652] [Q&A / Help] [answered]
Hi, I have configured Time-Series Bar Chart where I'm trying to display period next to period offset. Graph works but for some reason colours of original and offset bars are the same.
<img width="1111" alt="image" src="https://user-images.githubusercontent.com/39964247/132700317-76184ee0-6a02-4dde-abe8-7bfe14ffeae3.png">

Chart configuration:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/39964247/132700402-602d27ec-0bff-4611-9422-17365ef32319.png">
<img width="300" alt="image" src="https://user-images.githubusercontent.com/39964247/132700486-05ffc366-5065-43a1-8cc6-ceb300a125c7.png">

Thanks in advance!

**Comment by kamalkeshavani-aiinside** [comment, marked as answer]
In 1.3, I think this behavior is changed.
For earlier version, I would suggest to try 'label_colors' config in dashboard json metadata.

**Reply by zhorvat-lw** [reply]
Thank you!
